Bukkit/src/main/java/com/plotsquared/bukkit/placeholder/Placeholders.java

```diff
@@ -181,6 +183,40 @@ public Placeholders() {
             default:
                 break;
         }
+        if (identifier.startsWith("currentplot_localflag_")) {
+            final String[] splitValues = identifier.split("currentplot_localflag_");
```

Comment:
Sorry, I wanted to mention that yesterday already but things came but so I forgot about it. In that case, a simple `identifier.substring("currentplot_localflag_".length())` would be fine too (it would return an empty String `""` if no flag name is given)

Reply:
I made some benchmark (regex vs split vs substring) on my side and I saw much better performance with substring, so I took the liberty to also change other flags which use split.